docs/steamdeck-and-linux/playing-on-linux-legacy-guide.md

```diff
@@ -34,7 +34,6 @@ Now Steam will automatically launch Northstar when you hit play. Just launch the
 > **Note:** There is a current bug where the game would sometimes launch vanilla Titanfall 2 instead of Northstar. There is no universal fix for this, but people have reported changing Proton versions to _Proton 5.13_ or _Proton Experimental_ and deleting the Proton prefix folder (`Steam/steamapps/compatdata/1237970/`) could help resolve this issue. Using [Proton GE](https://github.com/GloriousEggroll/proton-ge-custom) has also been reported to resolve the issue.
 >
 > If you are still suffering from this bug, try running the game through Lutris. The bug doesn't seem to happen there
-
```

Comment:
What's with the random line removals in this file?

Reply:
honestly, this was meant to be removed in a different branch, accidentally removed it in this one and re-added it. it needs to be removed anyway because its super outdated and irrelevant since we've version gated way past that.

Reply:
this will be done in #113 so im gonna resolve this since the file is being nuked anyway

Reply:
> this will be done in #113 so im gonna resolve this since the file is being nuked anyway

This will cause merge conflicts though on the other branch depending on which (#111 or #113) is merged first. Imma try to resolve it myself rq.